actors/builtin/methods.go

```diff
@@ -70,7 +70,7 @@ var MethodsPower = struct {
 	UpdatePledgeTotal        abi.MethodNum
 	SubmitPoRepForBulkVerify abi.MethodNum
 	CurrentTotalPower        abi.MethodNum
-}{MethodConstructor, 2, 3, 4, 5, 6, 7, 8}
+}{MethodConstructor, 2, 3, 4, 5, 6, 8, 9}
```

Comment:
This is very subtle. Could you add a placeholder entry for deprecated method in the struct definition and keep the gapless sequence 1..9?